addon/components/influencer/profile-description/component.js

```diff
@@ -44,6 +44,10 @@ export default Component.extend(TooltipActivationMixin, {
 
   displayContact: and('profile.email', 'hasInbox'),
 
+  isLdaCompliant: computed('medias', 'medias.instagram.audience.legal_drinking_age', function() {
```

Comment:
no need to be binded to `media`

Reply:
And IMO you could just use a `computed.gte('medias.instagram.audience.legal_drinking_age', 0.716)`